app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,100 @@
+TASKS = ["be cool", "be calm", "be collected", "lose your ever-loving mind"]
+
+class TasksController < ApplicationController
+  def index
+    @tasks = Task.all
+  end
+
+  def show
+    task_id = params[:id]
+    @task = Task.find_by(id: task_id)
+    if @task.nil?
+      redirect_to tasks_path
+      return
+      head :not_found
+      return
```

Comment:
What's going on in this code? :) It reads: if the task is `nil`, redirect to the `tasks_path`, then `return`.... and then `head :not_found` and then another `return`? Remember, `return` statements ALWAYS make the code exit the method. Lines 14-15 will _never_ execute under any circumstance, so it's better to just delete these lines of code.